src/assets/Generator.Shared/ManagementPipelineBuilder.cs

```diff
@@ -14,9 +14,9 @@ namespace Azure.Core
 {
     internal static class ManagementPipelineBuilder
     {
-        public static HttpPipeline Build(TokenCredential credential, Uri endpoint, ClientOptions options)
+        public static HttpPipeline Build(TokenCredential credential, string scope, ClientOptions options)
```

Comment:
options:
1. double check all references to ensure no big impact
2. Be conservative, add an string overload.

Reply:
Currently, the MGMT sdks uses fixed version from eng/Directory.Build.Common.props: 
`<MgmtAutorestVersion>3.0.0-beta.20210504.1</MgmtAutorestVersion>`
In the meantime, the version we use in Azure.ResourceManager.Core is `3.0.0-beta.20210601.1`